src/core/ddsi/include/dds/ddsi/ddsi_security_omg.h

```diff
@@ -107,12 +151,326 @@ unsigned determine_publication_writer(const struct writer *wr);
  * @retval true   The proxy participant may be deleted.
  * @retval false  The proxy participant may not be deleted by this writer.
  */
-bool allow_proxy_participant_deletion(struct q_globals * const gv, const struct ddsi_guid *guid, const ddsi_entityid_t pwr_entityid);
```

Comment:
I had to look twice to realise that it tests whether deletion is allowed — were it named ``proxy_participant_deletion_allowed``, that would've been avoided. Perhaps you'd prefer to add "is" somewhere in the name, considering the functions declared further down in the file.

Reply:
Renamed.